evalml/model_understanding/graphs.py

```diff
@@ -1169,3 +1173,74 @@ def graph_t_sne(X, n_components=2, perplexity=30.0, learning_rate=200.0, metric=
     fig.update_layout(title='t-SNE', yaxis_zeroline=False, xaxis_zeroline=False)
 
     return fig
+
+
+def force_plot(pipeline, rows_to_explain, training_data, matplotlib=False):
```

Comment:
The issue mentions adding a "data method" to return the data needed to create the plot in json plot. Is that planned for a separate PR?

Reply:
I need to chat with @dsherry about what exactly that entails...I think I'll need some specifics but I planned to do that here.

Reply:
Our model understanding tools should be broken into two chunks:
* Data generation: given necessary inputs, return all the information required to visualize the results
* Visualization: call the data generation method, and use pyplot or similar to display the results

Separating these two steps is important because it means that the data generation method could be used to display the same information in a different UI.